curvefs/proto/mds.proto

```diff
@@ -95,6 +96,8 @@ message CreateFsRequest {
     required common.FSType fsType = 3;
     required FsDetail fsDetail = 4;
     required bool enableSumInDir = 5;
+    required string owner = 6;
+    required uint64 capacity = 7;
```

Comment:
This capacity seems not works after read the pr. How to limit the capacity of a fs?

Reply:
this commit just adds capacity and owner attr, not how to use them. 
i will call "create-fs --user xx --capacity xx" when develop curvefs csi.

quota limit and user mount/r/w restriction are follow-ups